kubetest/objects/deployment.py

```diff
@@ -34,12 +35,67 @@ class Deployment(ApiObject):
         'apps/v1beta2': client.AppsV1beta2Api,
     }
 
+    def __init__(self, *args, **kwargs):
+        super(Deployment, self).__init__(*args, **kwargs)
+        self._add_kubetest_labels()
+
     def __str__(self):
         return str(self.obj)
 
     def __repr__(self):
         return self.__str__()
 
+    def _add_kubetest_labels(self):
+        """Add a kubetest label to the Deployment object.
+
+        This allows kubetest to more easily and reliably search for and aggregate
+        API objects, such as getting the Pods for a Deployment.
+
+        The kubetest label key is "kubetest/<obj kind>" where the obj kind is
+        the lower-cased kind of the obj.
+        """
+        self.klabel_uid = str(uuid.uuid4())
+        self.klabel_key = 'kubetest/deployment'
+
+        # fixme: it would be nice to clean up this label setting logic a bit
+        #  and possibly abstract it out to something more generalized, but
```

Comment:
I think there is one extra space here

Reply:
the extra indentation here is for syntax highlighting - my editor will highlight the entire multi-line comment block as belonging to the fixme. I updated in my latest commit for it to be a 2-space indent to make it more clear